Strategies/TransitionToServices.md

```diff
@@ -0,0 +1,136 @@
+# A Strategy to transition IITB to a services oriented organization
+
+This document articulates a strategy and roadmap for moving IITB into a service provider by transforming IITB's Solution Delivery capability.
+
+The primary strategic goal of this transformation is to enable more rapid delivery of business-focused services at ESDC.
+
+This goal will be achieved through the establishment and execution of the following **7 key strategic initiatives**:
+
+1. Create an Enterprise Information Model
+2. Accelerate Cloud Adoption
+3. Invest in an Integration Layer (API-first approach)
+4. Implement DevOps
+5. Shift to Product Management
+6. Adopt a Bi-Modal Governance and Delivery
+7. Truly work in the open
```

Comment:
Truly work in the open -> Increase Volume of those working in the open

#actionoriented ;)

Reply:
Updated to: Transition to Working in the Open by Default